gemonlinedb/include/gem/onlinedb/OHv3Configuration.h

```diff
@@ -0,0 +1,411 @@
+#ifndef GEM_ONLINEDB_OHV3CONFIGURATION_H
+#define GEM_ONLINEDB_OHV3CONFIGURATION_H
+
+#include <algorithm>
+#include <array>
+#include <cstdint>
+#include <memory>
+
+#include "detail/OHv3ConfigurationGen.h"
+
+namespace gem {
+    namespace onlinedb {
+
+        // Forward declaration
+        class VFAT3ChipConfiguration;
+
+        /**
+         * @brief Describes the configuration of a version 3 optohybrid.
+         *
+         * @note Modifying objects of this class doesn't affect hardware.
+         */
+        class OHv3Configuration: public detail::OHv3ConfigurationGen
+        {
+        public:
+            /**
+             * @brief The number of wires in an HDMI cable.
+             */
+            static const constexpr std::size_t HDMI_WIRE_COUNT = 8;
+
+            /**
+             * @brief The number of VFATs connected to an OH.
+             */
+            static const constexpr std::size_t VFAT_COUNT = 24;
```

Comment:
May I suggest putting constants like this in `GEMConstants.h` under `gemutils`? It would be more clear why such a header is being `#include`d. Also, as it is now there's no need for `gemonlinedb` to depend on `gemhardware`.

Reply:
Yes, this is a good idea (and one that I have simmering had on the back burner for a while).
I hadn't implemented it for a variety of reasons, one of which was an investigation of how to minimally link certain libraries, but I think that all other misgivings I had have been either obviated or resolved.
If you have any *new* consts in this PR, feel free to create that file and put the new stuff there.
If you don't have any new consts, then just use the already defined consts.
Then I'll open another PR and migrate all the dependent code.